gitops/pac/setup.sh

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The pipelines-service Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+usage() {
+
+    printf "Usage: KUBECONFIG="path-to-kubeconfig" GITOPS_REPO="https://gitops.com/group/project" GIT_TOKEN="XXXXXXXXX" WEBHOOK_SECRET="YYYYYYYYYY" ./setup.sh\n\n"
+	
+    # Parameters
+    printf "The following parameters need to be passed to the script:\n"
+    printf "KUBECONFIG: the path to the kubeconfig file used to connect to the cluster where Pipelines as Code will be installed\n"
+    printf "GITOPS_REPO: the repository for which Pipelines as Code needs to be set up\n"
+    printf "GIT_TOKEN: personal access token to the git repository\n"
+    printf "WEBHOOK_SECRET: secret configured on the webhook used to validate the payload\n"
+    printf "OPERATOR_INSTALL (optional): the operator will be installed only if it is set to true. Pipelines as Code are installed together with the OpenShift Pipelines operator\n"
```

Comment:
It says that operator (I assume you're talking about [Red Hat Openshift Pipelines Operator](https://docs.openshift.com/container-platform/4.10/cicd/pipelines/installing-pipelines.html)) will be installed.

But in the operator_install function below at line 62, you're not installing the operator. You are manually installing just Pipelines as Code.

Reply:
Good point I will rephrase it

Reply:
Would you be rephrasing it in the other places too? I still see 4 occurrences of 'operator_install' in the PR.

Reply:
Sorry I should have been more diligent. It should be fixed now

Reply:
No worries. Thank you for the changes.